tests/fixtures/__init__.py

```diff
@@ -342,3 +343,15 @@ def temp_config(**overrides):
 
     with mock.patch.object(caikit.config.config, "_IMMUTABLE_CONFIG", local_config):
         yield local_config
+
+
+class TestServicerContext:
+    """
+    A dummy class for mimicking ServicerContext invocation metadata storage.
```

Comment:
Interesting. I was looking for a link to suggest as a citation of this interface and I found [this](https://github.com/grpc/grpc/blob/d038e0ed1970d8808e0f1e81acbefeb2f75f5941/src/python/grpcio_testing/grpc_testing/_server/_servicer_context.py#L19) instead. Makes me wonder if we should be using [grpcio-testing](https://pypi.org/project/grpcio-testing/) for all of this in our tests!

Reply:
Here's the citation link: https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/__init__.py#L1128